src/main/kotlin/com/ioki/lokalise/gradle/plugin/LokaliseExtension.kt

```diff
@@ -19,6 +19,8 @@ abstract class LokaliseExtension(
 
     val projectId: Property<String> = objects.property(String::class.java)
 
+    val pollUploadProcess: Property<Boolean> = objects.property(Boolean::class.java).convention(false)
```

Comment:
```suggestion
    val pollUploadProcess: Property<Boolean> = objects.property(Boolean::class.java).convention(true)
```

We don't want to introduce a breaking change 🙃